trackintel/model/trips.py

```diff
@@ -25,10 +28,11 @@ class TripsAccessor(object):
 
     The following assumptions are implemented
     
-        - All movement before the first and after the last activity is omitted
```

Comment:
I removed this because I think it is wrong, or misleading. As far as I could see, the triplegs before or after the first or last activity are also considered as trips, and are not ommitted. Please correct me if I'm wrong there

Reply:
No, I think you are right. It was different in the past, thanks for the update